docs/css/index.css

```diff
@@ -376,20 +387,91 @@ footer .footer__end-line .footer__code img {
     padding-left: 15px;
   }
 }
+.chapter .single-image-center-c {
+  text-align: center;
+  font-size: 0;
+  vertical-align: top;
+  margin-bottom: 35px;
+  max-width: calc(100vw - 40px);
+}
+.chapter .single-image-center-c > img {
+  max-width: calc(100vw - 40px);
+}
+.chapter.single-images-layout .chapter_inner {
+  width: 800px;
+}
+.chapter.single-images-layout h2.chapter__big-title:not(:first-child) {
+  margin-top: 60px;
+}
 .chapter .chapter__title-description {
```

Comment:
Let's add: 
```css
text-align: justify
```

It looks better that way.

Reply:
Done